### PR TITLE
Remove platform attribute and add labels

### LINKED ISSUE #185: Consider removing "platform" field in image and invocationImage
The image reference can point to either a single or multi-arch image in the registry. I think having the platform spec in bundle.json as well is confusing.

This remark might be nuanced for VM based scenarios, wdyt ?

### PR DESCRIPTION
**Pull request by jeremyrickard**
This PR removes the platform attribute from both `invocationImages` and `imageMap`. It is replaced with a free-form key/value attribute called "labels". 

Closes #185

### CODE REVIEW

File: schema/bundle.schema.json [hunk 1]
```diff
@@ -47,23 +47,18 @@
...
-              "description": "The architecture of the image (i386, amd64, arm32, arm64,...)",
-              "type": "string"
-            },
-            "os": {
-              "description": "The operating system of the image (linux, windows, darwin,...)",
-              "type": "string"
-            }
-          },
-          "type": "object"
+        "originalImage": {
```
Comment (silvin-lubecki) [inline schema/bundle.schema.json]: Was this part missing in the spec? Do you think you could add it in another PR? Or at least in a separate commit, just for a separation of concern reason ?

Comment (silvin-lubecki) [reply schema/bundle.schema.json]: Ok just dug it a little, @glyn removed `originalImage` field in this PR #218 
I guess that's a rebase artifact 😅

Comment (glyn) [reply schema/bundle.schema.json]: Correct. `originalImage` is no more and should be deleted from this PR.

Comment (jeremyrickard) [reply schema/bundle.schema.json]: Oops! yeah, rebase goof. Fixing.

**Comment by jeremyrickard** [comment]
@silvin-lubecki @glyn take a look again please!

**Comment by jeremyrickard** [comment]
OOPS i did a bad git thing. hold on

**Comment by glyn** [comment]
@silvin-lubecki Please remember to _squash_ and merge after you've approved.

**Pull request merged**